tests/unit/internal-session-test.js

```diff
@@ -385,7 +385,25 @@ describe('InternalSession', () => {
       return session.authenticate('authenticator');
     });
 
-    describe('when the authenticator resolves invaldiation', () => {
+    describe('when the authenticator resolves invalidation with params', () => {
```

Comment:
Sorry, oversaw that: this is the wrong text actually. This block is not actually about the authenticator resolving or rejecting, instead you'd want something like `describe('when arguments are passed', () => {`.